### PR TITLE
Person updates for DSTU2 and R4

### PR DESCRIPTION
**Pull request by amcginn**
Description
----
### DSTU2
* Updated field list URLs to 'https'
* "id" -> "ID" throughout
* Field descriptions
  * Updated info links to 'https'
* Moved disclaimer and errors to top
  * image
  * image
* Updated Search params table format/descriptions
  * image
  * image
* person-combine description
  * Removed quotes
  * Removed unsupported workflow statement 
  * image
  * image

### R4
* "id" -> "ID" throughout
* Removed "(Cerner alias Federated Person Principal)" from field list description
* Examples
  * Removed all but one identifier
  * Removed all but one name
  * Updated email and address text
* Moved disclaimer and errors to top
  * image
  * image
* Updated Search params table format/descriptions
  * image
  * image
* Updated Search example to use `identifier`
  * image
  * image
* person-combine description
  * Removed quotes
  * Removed unsupported workflow statement 
  * image
  * image
* Field descriptions
  * Corrected system URL
  * image
  * image

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [x] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by Someshnsn — APPROVED** [review]

File: content/millennium/dstu2/entities/person.md [hunk 1]
```diff
@@ -38,14 +44,14 @@ Search for Persons that meet supplied query parameters:
...
 ### Parameters
 
- Name         | Required?                                         | Type       | Description
---------------|---------------------------------------------------|------------|------------------------------------------------------------------------------------
- `_id`        | No, if populated all other parameters are ignored | [`token`]  | The logical resource id associated with the resource.
- `identifier` | Yes, or `_id`                                     | [`token`]  | The person identifier.  Example: `urn:oid:2.16.840.1.113883.3.13.6|01022228`
+ Name         | Required      | Type       | Description
+--------------|---------------|------------|------------------------------------------------------------------------------------
+ `_id`        | Conditionally | [`token`]  | The logical resource ID associated with the resource. This parameter is required if the `identifier` parameter is not used.
+ `identifier` | Conditionally | [`token`]  | The person identifier. This parameter is required if the `_id` parameter is not used.
```
Comment (kckeim) [inline content/millennium/dstu2/entities/person.md:50]: Did we want to have examples listed up in the table?  Most of our resources do.

File: content/millennium/r4/base/individuals/person.md [hunk 2]
```diff
@@ -55,22 +61,16 @@ Notes:
 
 #### Request
 
-    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Person?_id=12724067
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Person?identifier=urn%3Aoid%3A2.16.840.1.113883.6.1000%7C31577
```
Comment (kckeim) [inline content/millennium/r4/base/individuals/person.md:64]: Are you wanting to have the %AA codes here, or just colons and vertical pipes?  I know some systems have trouble passing the raw characters, but for readability, I think it looks better with how we had it in the DSTU2 resource with the colons displayed.

**Review by kckeim — APPROVED** [review]
Unchanged, so I couldn't flag it in-context: please HTTPS-ify the field_name_base_url in dstu2's person.yaml.  
Current:
field_name_base_url: http://hl7.org/fhir/DSTU2/person-definitions.html#Person

Beyond that, a couple comments is all.  Thanks for being patient while I took forever to get here.  Nothing from my side that would stop this from merging.

File: lib/resources/dstu2/person.yaml [hunk 3]
```diff
@@ -12,13 +12,13 @@ fields:
...
       description: The type used to determine which identifier to use for a specific purpose.
       terminology:
       - display: Identifier Type Codes
         system: http://hl7.org/fhir/identifier-type
-        info_link: http://hl7.org/fhir/DSTU2/valueset-identifier-type.html
+        info_link: https://hl7.org/fhir/DSTU2/valueset-identifier-type.html
       - display: v2 Identifier Type
         system: http://hl7.org/fhir/v2/0203
-        info_link: http://hl7.org/fhir/DSTU2/valueset-identifier-type.html
+        info_link: https://hl7.org/fhir/DSTU2/valueset-identifier-type.html
```
Comment (amcginn) [inline lib/resources/dstu2/person.yaml:24]:
```suggestion
        info_link: https://hl7.org/fhir/DSTU2/v2/0203/index.html
```

**Review by cldeitrich — APPROVED** [review]
[OHAIUAD-204315](https://jira3.cerner.com/browse/OHAIUAD-204315): Approved for style.

File: content/millennium/dstu2/entities/person.md (hunk 1 above)
Comment (amcginn) [reply content/millennium/dstu2/entities/person.md:50]:
```suggestion
 `_id`        | Conditionally | [`token`]  | The logical resource ID associated with the resource. This parameter is required if the `identifier` parameter is not used. Example: `_id=12345`
 `identifier` | Conditionally | [`token`]  | The person identifier. This parameter is required if the `_id` parameter is not used.  Example: `urn:oid:2.16.840.1.113883.3.13.6|01022228`
```

File: content/millennium/r4/base/individuals/person.md (hunk 2 above)
Comment (amcginn) [reply content/millennium/r4/base/individuals/person.md:64]: I've seen it both ways (🙄), but I agree
```suggestion
    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Person?identifier=urn:oid:2.16.840.1.113883.6.1000|31577
```

**Comment by Surya-Rnsit** [comment]
Validated after the code is published:
Screenshot 2024-02-02 at 3 40 44 PM
Screenshot 2024-02-02 at 3 38 27 PM
Screenshot 2024-02-02 at 3 38 03 PM

**Pull request merged**